Bio/phenotype/pm_fitting.py

```diff
@@ -54,8 +54,17 @@ def richards(x, A, u, d, v, y0):
 
     Proposed in Zwietering et al., 1990 (PMID: 16348228)
     """
-    y = (A * pow(1 + (v + (np.exp(1 + v) * np.exp((u / A) *
-                                                  (1 + v) * (1 + (1 / v)) * (d - x)))), -(1 / v))) + y0
+    y = (
+        A
+        * pow(
+            1
+            + (
+                v
+                + (np.exp(1 + v) * np.exp((u / A) * (1 + v) * (1 + (1 / v)) * (d - x)))
+            ),
+            -(1 / v),
+        )
+    ) + y0
```

Comment:
Well that is a long looking expression - but the original was already rather long. The single letter variable names are also less than ideal, but neither is black's fault.

Reply:
Yes, either way they are hard to read.